.github/workflows/slask-test.yml

```
@@ -0,0 +1,19 @@
+# yaml-language-server: $schema=https://json.schemastore.org/github-workflow.json
+
+name: Post an inline text message
+on:
+  push_request:
```

actionlint (1.7.12)

[error] unknown Webhook event "push_request". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)